extension-manifest-v3/src/pages/offscreen/urlfilter2dnr/index.js

```diff
@@ -1,4 +1,6 @@
-import convert from '/utils/dnr-converter.js';
+import { createDocumentConverter } from '/utils/dnr-converter.js';
+
+const convert = createDocumentConverter();
```

Comment:
This is a offscreen page which needs to create a document converter, no case where offscreen document should create another offscreen document